Framework/PythonInterface/plugins/algorithms/EnggFitPeaks.py

```diff
@@ -8,6 +8,7 @@
 
 from mantid.kernel import *
 from mantid.api import *
+from mantid.simpleapi import *
```

Comment:
In L13 in the new solution the simpleapi is already imported as mantid. Then, either the new import replaces the previous solution (I believe that'd be preferred ) and all references to simpleapi through mantid.algorithm_name should be refactored, or the new call to Fit should be done through mantid.fit.